includes/Commerce.php

```diff
@@ -53,7 +62,7 @@ public function get_default_google_product_category_id() {
 	 */
 	public function update_default_google_product_category_id( $id ) {
 
-		// TODO: implement
+		update_option( self::OPTION_GOOGLE_PRODUCT_CATEGORY_ID, is_string( $id ) ? $id : '' );
```

Comment:
I think that way we could end up storing `Array` as the default category or throwing an error if we try to cast a value that cannot be converted to string. If we were able to use a `string` type hint, the user would have to cast the value before calling the method, so I thought this would offer a similar "experience".

Reply:
@deivamagalhaes I'm on PTO this week, so to avoid blocking other changes, could you please take over this PR if you think additional changes are necessary?